docs/src/api/class-tracing.md

```diff
@@ -66,12 +72,18 @@ await context.Tracing.StopAsync(new()
 
 Start tracing.
 
+:::note
+This API records browser operations and network activity, but does not capture test assertions (`expect` calls).
```

Comment:
I think this also needs a call to action, which is "use trace" config file option.